### ISSUE TITLE
Update-Package -reinstall adds second <Private>True</Private>

### ISSUE THREAD

**Issue by Regenhardt** [NuGet/Home#10577]
## Details about Problem

Executing `Update-Package -reinstall -id System.Net.Http` adds a second `<Private>True</Private>` to its dependencies in the .csproj file:
![image](https://user-images.githubusercontent.com/10476293/108333531-54d0b680-71d1-11eb-985d-383c1f1424b7.png)

NuGet product used (NuGet.exe | Visual Studio | MSBuild.exe | dotnet.exe):
Visual Studio Package Manager Console

Product version: 
Visual Studio 16.8.4
NuGet Package Manager 5.8.1

Worked before? If so, with which NuGet version: 
Not as far as I know.

## Repro steps and/or sample project

Working in net462 project
Install System.Net.Http via Package Manager
Execute `Update-Package -reinstall -id System.Net.Http` from the VS Package Manager Console
Check the .csproj file

Tbh I haven't tried with other packages.

### Verbose Logs

As far as I can see, the Update-Package command doesn't provide a 
trigger for more detailed logs, and this command only outputs logs like

Retrieving package...
Added package... to package.config
Successfully installed... to {project}

**Comment by donnie-msft** [comment]
I was unable to repro this in a new framework 4.6.2 console app. 
The `<Private>` tag gets added when `CopyLocal=False`:
![image](https://user-images.githubusercontent.com/49205731/108605636-5fcb4880-7383-11eb-80b6-22bf72865e25.png)
To repro, set the CopyLocal to false, save, and you'll see the attribute added to your csproj.

My guess is you have some .targets or similar that is marking this assembly as private. You may want to try some MSBuild tooling like Binlog to understand why this is being added: https://docs.microsoft.com/en-us/visualstudio/msbuild/obtaining-build-logs-with-msbuild?view=vs-2019

**Comment by aortiz-msft** [comment]
Hi there. We are closing this issue since we didn't hear back from you, but definitely let us know if you can still reproduce this problem on the latest version of NuGet. Thank you for your feedback and happy coding!

**Comment by samrueby** [comment]
I believe that I reproduced it a few days ago.

**Comment by ddimkovikj** [comment]
Reproduced it as well using the command mentioned in OP.

Product version:
Visual Studio 16.9.5
NuGet Package Manager 5.9

**Comment by mlop3s** [comment]
Yes, this is still happening. You need to have .net Framework and .net Standard / Core / > 5.0 on the same solution. 
Visual Studio Version 17.4.3
NuGet Version: 6.1.0.106

**Comment by KhalilMohammad** [comment]
This is still happening. It is reproducible.
Visual Studio Version 17.19.1
NuGet Version: 6.9.1
